Makefile

```diff
@@ -31,40 +32,59 @@ all: docker
 gobuild:
 	./scripts/build false
 
+# create output directories
+.out-stamp:
+	mkdir -p ./out/test-artifacts ./out/cni-plugins
+	touch .out-stamp
+
+out: .out-stamp
+
 # Basic go build
 static:
 	./scripts/build
 
+BUILDER_IMAGE="amazon/amazon-ecs-agent-build:make"
```

Comment:
nit: can you move this to the top of the file?

Reply:
We talked about this offline and decided to keep variables near the targets that use them.